scout/constants/clinvar.py

```diff
@@ -102,3 +102,27 @@
     "practice_guideline",
     "reviewed_by_expert_panel",
 }
+
+# Models of inheritance are defined here:
+# https://ftp.ncbi.nlm.nih.gov/pub/GTR/standard_terms/Mode_of_inheritance.txt
+INHERITANCE_MODELS = [
```

Comment:
Could we preface that with like CLINVAR? One might be confused as GenMod uses a different set. (https://github.com/moonso/genmod/blob/master/docs/genetic-models.md)
Or name the clinvar one INHERITANCE_MODES and let the GenMod sphere keep "inheritance models". Then we have OMIM and HPO with same-same-but-diffrent. 🙄

Reply:
Sure! CLINVAR_INHERITANCE_MODELS is more precise